X16 Reference - Appendix G - Releases and compatibility.md

```diff
@@ -0,0 +1,103 @@
+
+# Appendix G: ROM releases and compatibility
+
+## Table Of Contents
+
+1. [ROM release overview](#rom-release-overview)
+2. [ROM release compatibility](#rom-release-compatibility)
+
+## ROM release overview
+
+This is a list of releases, using emulator releases as a reference point:
+
+Release 48 ("Cadmium")
+- Date: 2024-09-06
+- Emulator: https://github.com/X16Community/x16-emulator/releases/tag/r48
+- ROM: R48
+- Emulated VERA: 47.0.2
+
+Release 47 ("Roswell")
+- Date: 2024-03-30
+- Emulator: https://github.com/X16Community/x16-emulator/releases/tag/r47
+- ROM: R47
+- Emulated VERA: 47.0.0 (?)
+- Emulated SMC: 47.0.0
+
+Release 46 ("Winnipeg")
+- Date: 2023-11-06
+- Emulator: https://github.com/X16Community/x16-emulator/releases/tag/r46
+- ROM: R46
+- Emulated VERA/SMC: Not specified
+
+Release 45 ("Nuuk")
+- Date: 2023-10-17
+- Emulator: https://github.com/X16Community/x16-emulator/releases/tag/r45
+- ROM: R45
+- Emulated VERA: Some change, but version not clearly specified
+- Emulated SMC: 45.1.0
+
+Release 44 ("Milan")
+- Date: 2023-08-14
+- Emulator: https://github.com/X16Community/x16-emulator/releases/tag/r44
+- ROM: R44
+- Emulated VERA: Includes FX
+- Emulated SMC: Not specified
+
+Release 43 ("Stockholm")
+- Date: 2023-05-17
+- Emulator: https://github.com/X16Community/x16-emulator/releases/tag/r43
+- ROM: R43
+- Emulated VERA: Not specified
+- Emulated SMC: "r43" (BREAKING CHANGE regarding keyboard)
+
+Release 42 ("Cambridge")
+- Date: 2023-03-07
+- Highlights: First Community release
+- Emulator: https://github.com/X16Community/x16-emulator/releases/tag/r42
+- ROM: R42
+- Emulated VERA: Seem to be some updated vera features
+- Emulated SMC: Older than r43 due to breaking change
+
+Release 41 ("Marrakech")
+- Date: 2022-05-16
+- Emulator: https://github.com/commanderx16/x16-emulator/releases/tag/r41
+- ROM: R41
+- Emulated VERA and SMC: Not specified
```

Comment:
Release 41 was based on an earlier prototype where the PS/2 keyboard was connected directly to the VIA, so the ROM and SMC had breaking changes between R41 and R42.